adapter/repository/in-memory-simple/src/main/java/org/mobilitydata/gtfsvalidator/db/InMemoryGtfsSpecRepository.java

```diff
@@ -42,6 +44,10 @@
     private final GtfsSpecificationProto.CsvSpecProtos inMemoryGTFSSpec;
     private final Map<String, ParsedEntityTypeValidator> validatorByFilenameCache = new HashMap<>();
 
+    private static final String[] VALID_URL_SCHEMES = {"http", "https"};
+    private static final String VALID_COLOR_REGEX_PATTERN = "[0-9a-fA-F]{6}";
+    private static final String VALID_TIME_REGEXP_PATTERN = "([0-9][0-9]|[0-9]):[0-5][0-9]:[0-5][0-9]";
```

Comment:
IMHO something like "999:00:00" should be flagged as a warning, although it's not clear where the cutoff should be. FWIW, we used 29 hours as a cutoff for the GTFS-RT validator. This can be pushed to a future issue.

Reply:
Ok yes we shall have a notion for a user configurable suspicious threshold. I'll create an issue

Reply:
#91